Client/src/Pages/Auth/Login.jsx

```diff
@@ -19,99 +19,6 @@ import { logger } from "../../Utils/Logger";
 import "./index.css";
```

Comment:
`import Mail from "../../assets/icons/mail.svg?react";`

This icon import can be removed, it is no longer used